2.cellprofiler_processing/cputils.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+from pathlib import Path
+
+def run_cellprofiler(path_to_pipeline: str, path_to_output: str, path_to_images: str):
+    """Profile batch with CellProfiler (runs segmentation and feature extraction)
+
+    Args:
+        path_to_pipeline (str): path to the CellProfiler .cppipe file with the segmentation and feature measurement modules
+        path_to_output (str): path to the output folder for the .sqlite file
+        path_to_images (str): path to folder with IC images from specific plate
+    """
+    # currently, the output path in the command does not output correctly, so the directory of the terminal must be set to the output path
+    os.chdir(path_to_output)
+    # run CellProfiler on a plate
+    command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
+    os.system(command)
+
+def rename_sqlite_file(sqlite_file_path: pathlib.Path, plate: str):
+    """Rename the .sqlite file to be {plate}.sqlite as to differentiate between the files
+
+    Args:
+        sqlite_file_path (pathlib.Path): path to CellProfiler_output directory
+        plate (str): name of the plate that was run through CellProfiler
+    """
+    for sqlite_files in sqlite_file_path.iterdir():
+        # the CellProfiler pipeline hardcodes the .sqlite file name, so all files outputted are called "CFReT.sqlite"
```

Comment:
oh interesting. Have you checked if you can add the plate name and file extension in the `-o` flag in the CP run on line 16?

Reply:
@gwaybio 

I will look into this! This could be interesting if it can work!

Reply:
I could not get this to work unfortunately. It will just create a folder with that name. 

@gwaybio Let me know any advice you have to get this to work, if you have, no worries if not.

Reply:
it seems like it should be an option, but I am not seeing it here: https://github.com/CellProfiler/CellProfiler/blob/master/cellprofiler/__main__.py

I suppose your approach here is best, even though it seems like a fragile fix